### ISSUE TITLE
Dotted Circle is inserted between Gurmukhi Yakash and vowel sign

### ISSUE THREAD

**Issue by JelleBosmaMT** [harfbuzz/harfbuzz#1592]
Gurmukhi sign Yakash 0A75 represents a consonant and according to the original L2/06-037 Unicode proposal it should be after consonant and before the vowel sign. Low vowels are supposed to be pushed down when following the sign. There is some back and forth on Unicode about the shape of the character more recently, but I do not find a discussion about ordering.

In the Chrome browser it seems Harfbuzz expects a different order and inserts a dotted circle before the vowel. To the left Chrome and to the right Safari (which doesn't insert a dotted circle), that applies the ligature of yakash with the low vowel when the vowel follows the yakash. Windows matches Safari. Harfbuzz is the odd one out, as it seems to expect the yakash to follow vowel.
![image](https://user-images.githubusercontent.com/12729011/53083211-bc7a9b00-34fe-11e9-8c67-65d19e4d65c4.png)

This is the string trying out vowels before and after  ਠੵੇਠੇੵਠੇੵਠੵੇ ਠੂੵ ਠੵੂ 

Checked with Mac Chrome Version 72.0.3626.109

**Comment by jfkthame** [comment]
What font(s) are you using in each case?

**Comment by JelleBosmaMT** [comment]
The shown samples use an updated Noto Sans Gurmukhi that I am preparing for release. But the 2016 version gives the same result. Of course it is the software that inserts dotted circles, not the font.

**Comment by dscorbett** [comment]
The commit that broke this is 8bf4027d23318c7e1ff7fe9f5e7ad8b0380e5415. @behdad, what did you mean by “The only Indic CM is U+0A75 GURMUKHI SIGN YAKASH, which Uniscribe doesn't treat specially, and font designers design for that”? How are font designers supposed to design for that?

**Comment by behdad** [comment]
I must confess I don't remember why I removed that.  What I mean by that comment was that "people seem to be happy with not classifying this character in uniscribe, so we'll do the same" whereas the first part of that statement is probably not true.

I'm fine reverting this commit.  Though I have a vague memory that I had a good reason removing CMs.  Or maybe I just did it when I was simplifying Khmer shaper and noticed that CM is also mostly unused in the Indic shaper...
